firestore/spec/order_limit_data_spec.rb

```diff
@@ -0,0 +1,100 @@
+require_relative "../get_data.rb"
+require_relative "../order_limit_data.rb"
+require "rspec"
+require "google/cloud/firestore"
+
+def delete_collection collection_name:
```

Comment:
move delete_collection helper method into helpers.rb file within spec folder since this is used by multiple test files